core/client/hdfs/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java

```diff
@@ -211,6 +212,22 @@ public void hadoopShouldLoadFileSystemWhenConfigured() throws Exception {
     }
   }
 
+  /**
+   * Ensures that Hadoop loads the Alluxio file system with Zookeeper URI.
+   */
+  @Test
+  public void hadoopShouldLoadFileSystemWithZkUri() throws Exception {
+    org.apache.hadoop.conf.Configuration conf = getConf();
+
+    URI uri = URI.create(Constants.HEADER
```

Comment:
I think putting the literal "alluxio://zk@zkHost:2181/tmp/path.txt" would be easier to read. It's not like those constants would change any time. Leave it to you which way you prefer

Reply:
Changed to the easy reading format. In the future, if we want to change the format, those tests would fail and we will notice that the URIs need to be changed.